src/main/index.ts

```
@@ -72,3 +74,38 @@ app.on('window-all-closed', () => {
+
+    if (existsSync(seriesImagePath)) {
+      copyFileSync(seriesImagePath, destImagePath)
+    } else {
+      return { success: false, message: '이미지 경로가 존재하지 않습니다.' }
+    }
+
+    const meta = createMetaJson(seriesTitle, `image/${destImageName}`)
+    writeFileSync(metaPath, JSON.stringify(meta, null, 2), 'utf-8')
+
+    return { success: true, path: seriesPath }
+  } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(@typescript-eslint/no-unused-vars)